### PR TITLE
Metals 0.9.0

### PR DESCRIPTION
**Pull request by sideeffffect**
closes https://github.com/che-dockerfiles/che-sidecar-scala/pull/3

### CODE REVIEW

**Comment by sideeffffect** [comment]
@ericwill here we go :rocket:

File: .github/workflows/build-publish.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+name: Build and push container
+
+on:
+  push:
+    branches:
```
Comment (ericwill) [inline .github/workflows/build-publish.yaml:14]: The branch name should be here, just like on the 1.8.6 branch. Otherwise it's good to go.

**Review by ericwill — CHANGES_REQUESTED** [review]

File: .github/workflows/build-publish.yaml (hunk 1 above)
Comment (sideeffffect) [reply .github/workflows/build-publish.yaml:14]: sure thing, it's back

**Review by ericwill — APPROVED** [review]

File: .github/workflows/build-publish.yaml (hunk 1 above)
Comment (sideeffffect) [reply .github/workflows/build-publish.yaml:14]: omg, I put `1.8.6` there :joy: 
thanks for fixing that up :+1:

**Pull request merged**